website/docs/sdk-reference/php.md

```diff
@@ -177,25 +303,61 @@ You can use the following caching options:
 ## Logging
 The SDK uses the PSR-3 `LoggerInterface` for logging, so you can use any implementor package like [Monolog](https://github.com/Seldaek/monolog).
 ```php
-
 $client = new \ConfigCat\ConfigCatClient("#YOUR-SDK-KEY#", [
-    'logger' => new \Monolog\Logger("name"),
+    \ConfigCat\ClientOptions::LOGGER => new \Monolog\Logger("name"),
 ]);
 ```
-
-## HTTP Client (Proxy)
-The SDK uses [Guzzle](http://docs.guzzlephp.org/en/stable/index.html) for the underlying HTTP calls and its request options are available to customize through the SDK options like:
+You can change the verbosity of the logs by setting the `log-level` configuration option.
 ```php
+$client = new \ConfigCat\ConfigCatClient("#YOUR-SDK-KEY#", [
+    \ConfigCat\ClientOptions::LOG_LEVEL => \ConfigCat\Log\LogLevel::INFO
+]);
+```
+
+The following levels are used by the ConfigCat SDK:
+
+| Level      | Description                                                                             |
+| ---------- | --------------------------------------------------------------------------------------- |
+| `NO_LOG`   | Turn the logging off.                                                         |
+| `ERROR`    | Only error level events are logged.                                                     |
+| `WARNING`  | Errors and Warnings are logged.                                                         |
+| `INFO`     | Errors, Warnings and feature flag evaluation is logged.                                 |
+| `DEBUG`    | All of the above plus debug info is logged. Debug logs can be different for other SDKs. |
 
+> The default level is `WARNING`. With `NO_LOG` you can turn the logging completely off.
```

Comment:
this line blends in with the rest of the text. I suggest removing or coloring.

Reply:
Done